src/components/FormFields/Custom/DataIntegrityChecksField.module.css

```diff
@@ -1,23 +1,38 @@
 .transfer {
-    margin-top: 8px;
+    margin-block-start: var(--spacers-dp8);
 }
 
 .transferOption {
+    display: flex;
```

Comment:
I changed this to flex, mostly because I prefer using `gap` instead of `margins` for spacing between the header and other stuff.